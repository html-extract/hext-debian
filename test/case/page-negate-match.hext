<a href^="https:"! class:class />
