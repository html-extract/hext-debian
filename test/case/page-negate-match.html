<a class="yes1"></a>
<a href class="yes2"></a>
<a href="#a" class="yes5"></a>
<a href="http" class="yes3"></a>
<a href="https" class="yes4"></a>
<a href="https:" class="no1"></a>
<a href="https://asd" class="no2"></a>
