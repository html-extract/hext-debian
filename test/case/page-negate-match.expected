{"class":"yes1"}
{"class":"yes2"}
{"class":"yes3"}
{"class":"yes4"}
{"class":"yes5"}
